client/packages/common/src/intl/utils/IntlUtils.ts

```diff
@@ -62,28 +49,47 @@ export const IntlUtils = {
       throw new Error(`Language '${language}' not supported`);
     }
     return 'en';
-  },
-  languageOptions,
-  getLanguageName: (language: string) =>
-    languageOptions.find(option => option.value === language)?.label,
-  getUserLocale: (username: string) => {
+  })();
+
+  const currentLanguageName = languageOptions.find(
+    option => option.value === language
+  )?.label;
+
+  const getLocaleCode = (language: LanguageType) => parseLanguage(language);
+
+  const getUserLocale = (username: string) => {
     const locales = LocalStorage.getItem('/localisation/locale');
     return !!locales ? locales[username] : undefined;
-  },
-  setUserLocale: (username: string, locale: SupportedLocales) => {
+  };
+
+  const setUserLocale = (username: string, locale: SupportedLocales) => {
     const locales = LocalStorage.getItem('/localisation/locale') ?? {};
     locales[username] = locale;
     LocalStorage.setItem('/localisation/locale', locales);
-  },
-  useLocalisedFullName: () => {
-    const { i18n } = useTranslationNext();
-    const { language } = i18n;
-    return (firstName: StringOrEmpty, lastName: StringOrEmpty) =>
-      getFullName(language, firstName, lastName);
-  },
+  };
+
+  const getLocalisedFullName = (
+    firstName: StringOrEmpty,
+    lastName: StringOrEmpty
+  ) => getFullName(language, firstName, lastName);
+
+  return {
+    // TODO: When the server supports a query to find the deployments
+    // default language, use a query to fetch it.
+    defaultLanguage: 'en',
```

Comment:
I couldn't find a usage of this - wondering why it is added?
The default language would be more useful in the IntlContext, where the `fallbackLng` is set. Though I doubt that is helpful either.
Also not sure about having a TODO here for fetching a default from the API - that is better at the host / app level and not in a helper function like this.

Reply:
Yeah, it's not actually being used at all, but it was there in the original, so I left it in:
```ts
  // TODO: When the server supports a query to find the deployments
  // default language, use a query to fetch it.
  useDefaultLanguage: (): SupportedLocales => {
    return 'en';
  },
```

I'll get rid of it, keep it simple. :) 19382297d